### PR TITLE
style(layout): change font-family

### PR DESCRIPTION
**Pull request by Aharonla**

### CODE REVIEW

File: src/components/TeamMember.js [hunk 1]
```diff
@@ -5,14 +5,14 @@ import { GatsbyImage, getImage } from 'gatsby-plugin-image';
 
 function TeamMember({ social, name, role, image }) {
   return (
-    <div className="flex flex-col items-center text-tezos-blue museo p-1 lg:w-1/3 md:w-1/3 w-full mb-4">
+    <div className="flex flex-col items-center text-tezos-blue mono p-1 lg:w-1/3 md:w-1/3 w-full mb-4">
```
Comment (chiptus) [inline src/components/TeamMember.js:8]: just `mono` or `museo` doesn't do anything. I guess it's a mistake. remove it

**Review by chiptus — CHANGES_REQUESTED** [review]
just one change, LGTM otherwise

File: src/components/TeamMember.js (hunk 1 above)
Comment (Aharonla) [reply src/components/TeamMember.js]: it was a mistake, removed

**Comment by chiptus** [comment]
decided to close

**Pull request closed without merging**